src/components/overlays/quiz-overlay.js

```diff
@@ -144,7 +144,7 @@ class QuizOverlay extends BindingHelpersMixin(IocRequesterMixin(LocalizationMixi
       /** The first question that has to be shown in the current second. Used for displaying the overlay. Should probably be computed, that didn't work though */
       _currentQuestion: {
         type: Object,
-        value: null,
+        computed: '_getCurrentQuestion(_currentQuestions)',
```

Comment:
Ah I see the problem you are facing with the computed properties. You just observe value changes of the `_currentQuestions` property here. However, the property is not changed, but the contained array is modified. Therefore, you need to observe [array mutations](https://www.polymer-project.org/3.0/docs/devguide/observers#array-observation). 

Using `_getCurrentQuestion(_currentQuestions.splices)` should work.

Reply:
Yup, that works, thanks. I'm pretty sure I read through these docs before but I didn't see that. Guess I had the wrong version open.